ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -17,14 +17,12 @@ class ProgressBar(object):
             [default: '{desc}[{n_fmt}/{total_fmt}] {percentage:3.0f}%|{bar}{postfix} [{elapsed}<{remaining}]'].
             Set to ``None`` to use ``tqdm`` default bar formatting: '{l_bar}{bar}{r_bar}', where
             l_bar='{desc}: {percentage:3.0f}%|' and
-            r_bar='| {n_fmt}/{total_fmt} [{elapsed}<{remaining}, '
-              '{rate_fmt}{postfix}]'
-            Possible vars: l_bar, bar, r_bar, n, n_fmt, total, total_fmt,
-              percentage, rate, rate_fmt, rate_noinv, rate_noinv_fmt,
-              rate_inv, rate_inv_fmt, elapsed, remaining, desc, postfix.
-            Note that a trailing ": " is automatically removed after {desc}
-            if the latter is empty.
-        **tqdm_kwargs: kwargs passed to tqdm progress bar
+            r_bar='| {n_fmt}/{total_fmt} [{elapsed}<{remaining}, {rate_fmt}{postfix}]'. For more details on the
+            formatting, see `tqdm docs <https://tqdm.github.io/docs/tqdm/>`_.
+        **tqdm_kwargs: kwargs passed to tqdm progress bar.
+            By default, progress bar description displays "Epoch [5/10]". If tqdm_kwargs defines
+            `desc`, e.g. "Predictions", than the description is "Predictions [5/7]" if number of epochs is more
```

Comment:
can we change this to "Predictions [5/10]"? Don't want anyone getting confused over why it went from "5/10" to "5/7"

Reply:
Yes, you are right. It is unclear what are these numbers 5, 7 and 10. I'll improve this.

Reply:
@vfdev-5 looks perfect now! I'll let you merge when you are ready.